spec/Creator/MassInvoicesCreatorSpec.php

```diff
@@ -34,19 +34,19 @@ function it_requests_invoices_creation_for_multiple_orders(
         OrderInterface $secondOrder,
         OrderInterface $thirdOrder
     ): void {
-        $firstOrder->getNumber()->willReturn('0000001');
-        $secondOrder->getNumber()->willReturn('0000002');
-        $thirdOrder->getNumber()->willReturn('0000003');
+        $firstOrder->getNumber()->willReturn('000001');
+        $secondOrder->getNumber()->willReturn('000002');
+        $thirdOrder->getNumber()->willReturn('000003');
```

Comment:
To ease the reviewing of your PR by Sylius core team members, can you revert it by adding the missing zero ?